### PR TITLE
Results

### PR DESCRIPTION
**Pull request by chanchal16**
- Created results page.
- shows score with quiz summary.
- shows the correct answer if the selected option is wrong.

Preview link: https://stirring-puppy-ad6f18.netlify.app/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ function App() {
           <Route path='/' element={<Categories/>} />
           <Route path='rules' element={<Rules/>}/>
           <Route path='questions' element={<Questions/>}/>
+          <Route path='results' element={<Results/>}/>
```
Comment (vidhi-thakur) [inline src/App.jsx:15]: You can also add a page for 404

File: src/pages/Results.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useQuestions } from '../contexts/QuestionsContext';
+import '../styles/results.css';
+
+export function Results() {
+  const {quizState,questions} = useQuestions();
+  const{score,results} = quizState;
+  console.log('results',quizState)
```
Comment (vidhi-thakur) [inline src/pages/Results.jsx:8]: please remove the console logs

File: src/pages/Results.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <div className='result-container'>
+      <h4 className='h6 center-text'>Your score:  
+        <span className='score'>{score}/{questions.length * 10}</span>
+      </h4> 
+      <div className='result-summary'>
+        {
+          results?.map(({ques,answer},index)=>(
+            <div className='result-card'>
```
Comment (vidhi-thakur) [inline src/pages/Results.jsx:17]: can create a separate component here for result card

**Review by vidhi-thakur — COMMENTED** [review]
Approved with comments
- your file structure looks good 
- code is readable
Good work 👍

**Review by vidhi-thakur — APPROVED** [review]

File: src/App.jsx (hunk 1 above)
Comment (chanchal16) [reply src/App.jsx:15]: Yup will add it

**Review by sandeepkumardev-zz — COMMENTED** [review]
looks good for me. 
code structure is also good.

**Pull request merged**